### ISSUE TITLE
Data inquiry

### ISSUE THREAD

**Issue by xuansuyang** [cailab-tamu/scTenifoldXct#1]
First and foremost, I want to express my sincere gratitude for developing this tool. I am interested in using your sample data to perform code testing on my local machine. I was wondering if it would be possible for you to kindly share a package containing the files used in your examples? Thank you in advance for your consideration.

**Comment by yjgeno** [comment]
Hi @xuansuyang, thanks for your interest in our paper. You can access all the relevant data on GEO. For example, both raw and processed data for our first case study on inflammatory skin have been placed at GSE147424. If you require a small example of toy data for a quick overview, I'm happy to share it with you. Could you provide your email address?

**Comment by xuansuyang** [comment]
Yes, I need a small example of toy data for a quick overview. My email is xuansuyang@foxmail.com. Thank you for your reply and wish you a happy life.

**Comment by yjgeno** [comment]
Hi @xuansuyang I just contacted you and uploaded the data to this repo.
